com.github.dweymouth.supersonic.yml

```diff
@@ -0,0 +1,212 @@
+# example flatpaks:
+# https://github.com/search?q=org%3Aflathub+golang+&type=code&p=2
+# https://github.com/search?q=org%3Aflathub+GLFW+&type=code
+#
+# followup on:
+# https://discourse.flathub.org/t/supersonic-lightweight-cross-platform-desktop-client-for-subsonic-music-servers/3984/2
+# https://github.com/dweymouth/supersonic/issues/86
+app-id: com.github.dweymouth.supersonic
```

Comment:
i was under the impression that the point of the app-id was that it was a space the upstream controlled. since there was no github.io site, i thought it was better to use github.com since that is where the site actually is, but i'll trust you on this and i changed it to io.github. :)

Reply:
actually `io.github.USER_OR_ORG` is always granted. Also this allow web based verification schemes.